examples/includes.cxx

```diff
@@ -0,0 +1,21 @@
+#include <iostream>
```

Comment:
Testing out the individual includes (e.g. forall and sequential, without all of the rest). I can get rid of it.